cpp/CMakeLists.txt

```diff
@@ -443,7 +443,7 @@ if(BUILD_CUML_CPP_LIBRARY)
     src/metrics/v_measure.cu
     src/pca/pca.cu
     src/randomforest/randomforest.cu
-    src/random_projection/rproj.cu
+    src/rproj/rproj.cu
```

Comment:
Why this new folder/name change?

Reply:
Sorry, for late answer (earlier message apparently wasn't sent). Basically, it's just for coherence with the rest of the code. Models in the cpp layer are in directories named after the models' diminutive names. In this case the files already use the `rproj` diminutive.